### PR TITLE
Changelog v1.14.1

### PR DESCRIPTION
**Pull request by 6543**
as title

### CODE REVIEW

**Review by lunny — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,35 @@ This changelog goes through all the changes that have been made in each release
...
+  * Fix wiki clone urls (#15430) (#15431)
+  * Fix dingtalk icon url at webhook (#15417) (#15426)
+  * Standardise icon on projects PR page (#15387) (#15408)
+* ENHANCEMENTS
+  * Add option to skip LFS/attachment files for `dump` (#15407) (#15492)
+  * Clone panel fixes (#15436)
+  * Use semantic dropdown for code search query type (#15276) (#15364)
+* BUILD
+  * Build go-git variants for windows (#15482) (#15487)
+  * Quick template fix for built-in SSH server in admin config (#15464) (#15481)
```
Comment (jolheiser) [inline CHANGELOG.md]: Woops, wrong label on PR. This is `bug` (in `ui`)

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,35 @@ This changelog goes through all the changes that have been made in each release
...
+  * Lock down build-images dependencies (Partial #15479) (#15480)
+* MISC
+  * Fix potential copy lfs records failure when fork a repository (#15441) (#15485)
+
+
```
Comment (techknowlogick) [inline CHANGELOG.md]:
```suggestion
```
only one trailing line

File: CHANGELOG.md (hunk 1 above)
Comment (6543) [inline CHANGELOG.md]:
```suggestion
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,35 @@ This changelog goes through all the changes that have been made in each release
...
 
+## [1.14.1](https://github.com/go-gitea/gitea/releases/tag/v1.14.1) - 2021-04-15
+
+* BUGFIXES
+  * Github Migration ignore rate limit, if not enabled (#15490) (#15495)
+  * Use subdir for URL (#15446) (#15493)
+  * Query the DB for the hash before inserting in to email_hash (#15457) (#15491)
+  * Ensure review dismissal only dismisses the correct review (#15477) (#15489)
+  * Use index of the supported tags to choose user lang (#15452) (#15488)
+  * Fix wrong file link in code search page (#15466) (#15486)
```
Comment (6543) [inline CHANGELOG.md:16]:
```suggestion
  * Fix wrong file link in code search page (#15466) (#15486)
  * Quick template fix for built-in SSH server in admin config (#15464) (#15481)
```

**Comment by 6543** [comment]
@jolheiser done

**Review by zeripath — APPROVED** [review]

**Review by 6543 — COMMENTED** [review]
LGTM

**Review by techknowlogick — DISMISSED** [review]
blocking so we can get https://github.com/go-gitea/gitea/pull/15447 in this release

File: CHANGELOG.md [hunk 4]
```diff
@@ -4,6 +4,34 @@ This changelog goes through all the changes that have been made in each release
...
+  * Standardise icon on projects PR page (#15387) (#15408)
+* ENHANCEMENTS
+  * Add option to skip LFS/attachment files for `dump` (#15407) (#15492)
+  * Clone panel fixes (#15436)
+  * Use semantic dropdown for code search query type (#15276) (#15364)
+* BUILD
+  * Build go-git variants for windows (#15482) (#15487)
+  * Lock down build-images dependencies (Partial #15479) (#15480)
+* MISC
+  * Fix potential copy lfs records failure when fork a repository (#15441) (#15485)
```
Comment (6543) [inline CHANGELOG.md:35]:
```suggestion
  * Performance improvement for list pull requests (#15447) (#15500)
  * Fix potential copy lfs records failure when fork a repository (#15441) (#15485)
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -4,6 +4,34 @@ This changelog goes through all the changes that have been made in each release
 without substantial changes to our git log; to see the highlights of what has
 been added to each release, please refer to the [blog](https://blog.gitea.io).
 
+## [1.14.1](https://github.com/go-gitea/gitea/releases/tag/v1.14.1) - 2021-04-15
+
+* BUGFIXES
+  * Github Migration ignore rate limit, if not enabled (#15490) (#15495)
```
Comment (6543) [inline CHANGELOG.md:11]:
```suggestion
  * Fix bug clone wiki (#15499) (#15502)
  * Github Migration ignore rate limit, if not enabled (#15490) (#15495)
```

**Comment by lunny** [comment]
@techknowlogick I think this is ready.

**Pull request merged**